packages/playwright-core/src/server/injected/roleUtils.ts

```diff
@@ -972,6 +1004,8 @@ let cacheAccessibleName: Map<Element, string> | undefined;
 let cacheAccessibleNameHidden: Map<Element, string> | undefined;
 let cacheAccessibleDescription: Map<Element, string> | undefined;
 let cacheAccessibleDescriptionHidden: Map<Element, string> | undefined;
+let cacheAccessibleErrorMessage: Map<Element, string> | undefined;
```

Comment:
these caches should be reset similar to other in beginAriaCaches/endAriaCaches below

Reply:
I've updated the code to include the missed part. Thanks for the nice feedback. :)